SECURITY.md

```diff
@@ -1,7 +1,9 @@
 To report a security issue, please use the "Report a vulnerability" button on
-this page. Our vulnerability management team will respond within 3 working days
-of your report. If the issue is confirmed as a vulnerability, we will open a
-Security Advisory. This project follows a 90 day disclosure timeline.
+GitHub's Security tab for `jj`'s main repo, also accessible at
+<https://github.com/jj-vcs/jj/security/advisories>. Our vulnerability management
```

Comment:
What do you think about simplifying to "... button under [Advisories](https://github.com/jj-vcs/jj/security/advisories)."?

Reply:
How about this?

Because of my (armchair) concern about phishing, I also wanted to give people enough breadcrumbs to be able to find the right place without using the link, in case they feel (rightly or wrongly) worried about using the link.

Reply:
I was thinking that the reader was accessed this doc via https://github.com/jj-vcs/jj/security/policy. I think you're saying that it may be useful to give the full instructions in case they find this doc in the source tree. Makes sense.